include/staff/templates/thread-entries.tmpl.php

```diff
@@ -34,6 +34,8 @@
         // TODO: Consider adding a date boundary to indicate significant
         //       changes in dates between thread items.
         foreach ($buckets as $entry) {
+            $entry = ThreadEntry::lookup($entry->id);
```

Comment:
Why lookup? You already have entry object. right?

Reply:
It's an AnnotatedModel__ThreadEntry.

Reply:
I thought it delegates to the object.

Reply:
It ended up the way it did because of the sortEntries function. Buckets had to end up being an array of thread entries so that the thread entries show up in the right order. When you return them as an array and then loop through them, it says they're in the AnnotatedModel__ThreadEntry class.